dhall-json/src/Dhall/JSON.hs

```diff
@@ -213,31 +213,31 @@ module Dhall.JSON (
     ) where
 
 import Control.Applicative (empty, (<|>))
-import Control.Monad (guard)
 import Control.Exception (Exception, throwIO)
-import Data.Aeson (Value(..), ToJSON(..))
+import Control.Monad (guard)
+import Data.Aeson (ToJSON (..), Value (..))
 import Data.Maybe (fromMaybe)
-import Data.Monoid ((<>), mempty)
+import Data.Monoid (mempty, (<>))
 import Data.Text (Text)
 import Data.Text.Prettyprint.Doc (Pretty)
 import Data.Void (Void)
-import Dhall.Core (Binding(..), DhallDouble(..), Expr)
-import Dhall.Import (SemanticCacheMode(..))
-import Dhall.Map (Map)
+import Dhall.Core (Binding (..), DhallDouble (..), Expr)
+import Dhall.Import (SemanticCacheMode (..))
 import Dhall.JSON.Util (pattern V)
+import Dhall.Map (Map)
 import Options.Applicative (Parser)
 import Prelude hiding (getContents)
 
-import qualified Data.Aeson                            as Aeson
-import qualified Data.Foldable                         as Foldable
-import qualified Data.HashMap.Strict                   as HashMap
+import qualified Data.Aeson as Aeson
+import qualified Data.Foldable as Foldable
+import qualified Data.HashMap.Strict as HashMap
```

Comment:
It's just a style thing, but this "unalignment" IMHO slightly hurts readability by making the qualified names slightly harder to find.

Reply:
By setting on .stylish-haskell.yaml file `pad_module_names: true` I get your desired result:
```diff
- import qualified Data.Aeson as Aeson
- import qualified Data.Foldable as Foldable
- import qualified Data.HashMap.Strict as HashMap
+ import qualified Data.Aeson                            as Aeson
+ import qualified Data.Foldable                         as Foldable
+ import qualified Data.HashMap.Strict                   as HashMap
```
But other non-qualified import lists gets affected:
```diff
- import Control.Applicative (empty, (<|>))
- import Control.Exception (Exception, throwIO)
- import Control.Monad (guard)
+ import Control.Applicative       (empty, (<|>))
+ import Control.Exception         (Exception, throwIO)
+ import Control.Monad             (guard)
```

Let me know if you're okay with this or not so I deletee .stylish-haskell yaml and defintely turn off my auto-formatting plugin :sweat_smile:

Reply:
That looks pretty nice to me! :)

@Gabriel439 what do you think?

Reply:
Are the qualified and non-qualified imports padded separately?

Reply:
@Gabriel439 to answer your question, with current settings (`align: group`) it formats everything in group of imports, using longest line as a basis for that. (example below)

I'm doing some tests and auto-formatting of imports is really opinionated.

The only way to automatically avoid non-qualified import module names to have the padding of qualified space is by having them in separate groups. For example,

```haskell
import A.B (fun1, fun2)
import C.D.E (fun3, fun4)

import qualified F.G as FG
import qualified G.H.C as GHC
import H.I (fun5, fun6)
```
will format to this:
```haskell
import A.B   (fun1, fun2)
import C.D.E (fun3, fun4)

import qualified F.G   as FG
import qualified G.H.C as GHC
import           H.I   (fun5, fun6)
```

There is no available setting related to avoid this result. But if we place imports this way:

```haskell
import A.B (fun1, fun2)
import C.D.E (fun3, fun4)
import H.I (fun5, fun6)

import qualified F.G as FG
import qualified G.H.C as GHC
```

stylish-haskell will format them this way:

```haskell
import A.B   (fun1, fun2)
import C.D.E (fun3, fun4)
import H.I   (fun5, fun6)

import qualified F.G   as FG
import qualified G.H.C as GHC
```

which is the desired format. So we will need to have qualified and non-qualified imports separate to get our desired result

Reply:
Alright, if the imports are grouped for alignment purposes, then I prefer `pad_module_names: true`

Reply:
Ok cool, I'll update it then.